src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -69,7 +69,12 @@
     <EmbeddedResource Include="Debugging\Resources\*.*" LogicalName="Debugging/%(FileName)%(Extension)" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\..\Analyzers\CSharp\Tests\UseVarTestExtensions.cs" Link="CodeRefactorings\UseVarTestExtensions.cs" />
+    <Compile Include="..\..\Features\CSharpTest\CodeActions\AbstractCSharpCodeActionTest.cs" Link="CodeActions\AbstractCSharpCodeActionTest.cs" />
```

Comment:
do the rest of the tests implementing `AbstractCSharpCodeActionTest` in editorfeatures make use of actual editor packages?  Or are you planning on moving those down later as well (similar q for the other linked files)